docs/appendix/terminology.md

```diff
@@ -0,0 +1,136 @@
+# CernVM-FS Terminology
+
+An overview of terms used in the context of CernVM-FS, in alphabetical order.
+
+---
+
+## Catalog { #catalog }
+
+A **catalog** of a CernVM-FS [repository](#repository) is a table that lists files and directories along with the corresponding
+metadata (sizes, timestamps, etc.).
+
+Catalogs can be *nested*: subtrees of the [repository](#repository) may have their own catalog.
+
+For more information on the *catalog* concept, see [the CernVM-FS
+documentation](https://cvmfs.readthedocs.io/en/stable/cpt-details.html#file-catalog).
+
+---
+
+## CernVM { #cernvm }
```

Comment:
Perhaps add a sentence here along the lines of: "While originally developed in conjunction, the CernVM File System today is a product completely independent from the virtual appliance."

Reply:
added in cf17e20